### DISCUSSION TITLE
permission中找不到“PRIVATE”"GROUP"等相关参数

### DISCUSSION THREAD

**Discussion by Viottery** [nonebot/discussions#29] [问答] [answered]
在permission的代码中只找到了
MESSAGE 
NOTICE 
REQUEST 
METAEVENT
USER
SUPERUSER的相应内容。想问一下private和group等matcher在哪里调用

**Comment by mnixry** [comment, marked as answer]
如果用了cqhttp adapter的话
应该是在`nonebot.adapters.cqhttp`里

**Reply by Viottery** [reply]
好的！我去看看！

**Reply by Viottery** [reply]
已解决，谢谢！
